.github/workflows/cicd.yaml

```diff
@@ -204,6 +232,18 @@ jobs:
           cache: npm
       - name: Install dependencies
         run: npm install
+      - name: Download api-reference docs 
+        # download docs to python-sdk/api-reference-docs
+        uses: actions/download-artifact@v2
```

Comment:
This is a nice trick, but relying on github action tools makes it difficult to test locally.  We should put all this into a bash script or make target so we can preview a full docs build.

Reply:
Added a make target for this: `make docs-comprehensive`